files-community/Files
# Fix: Fixed an issue where it wasn't possible to edit some file permissions

### Description

Fix an issue which has been reported.

### Motivation and Context

I've been aware of that there're a lot of lack of function. As a part of improving, I fix an issue where some security functionalities are missing.

### Validation

- If you have a file with no ACEs
  - Remove all ACEs
  - Open its properties and see Security again
  - You can see a message that is telling there's no ACE
- If you have a file whose owner is not current user
  - Change owner to whatever that is not you like SYSTEM, Administrators
  - Open its properties and see Security again
  - You can see a message that is telling you do not have Read Permission access rights and to see ACEs, change owner.

### Removal & Its Rationale

- Removal of functionalities
  - I removed a functionality of changing access control type
  - I removed a functionality of changing access control flags
  - I removed a functionality of changing inheritance flags
  - I removed a functionality of converting inherited ACE to explicit one
  - I removed a functionality of removing inherited ACE
- Rationale of that removal
  - I will add back them in a further PR. Those functionalities are not at work long time before and so I decided those are might as well not to be.
- Removal of this message
- Rationale
  - File Explorer can see ACEs with admin privileges even if the current user doesn't have Read Permission. However, WinUI3 app cannot elevate current instance without no restarting, so there's no meaning that Continue button exists. In this case, in Files, Files suggest that you should take ownership of the object even if the current user can see ACEs with admin privileges like File Explorer do when even admin privilege doesn't work (for example, the owner is SYSTEM or something else that doesn't belong to Admin group).
  ![image](https://user-images.githubusercontent.com/62196528/235356462-0df286fd-d347-443f-9295-24649ef66d9f.png)


### PR Checklist
  - Related #11551 
  - Related #11550 
  - Related #11549
  - Related #11930 
  - Closes #12255
  - Related #11471
  - See Validation section.

### Screenshots

Before (File Explorer or previous Files)|After
---|---
![image](https://user-images.githubusercontent.com/62196528/234718315-b5e0e8b9-49fa-4a19-a793-8cc68a3790d2.png)|![image](https://user-images.githubusercontent.com/62196528/235287631-dbec3754-5485-4d53-89c5-a96bfe83c0ce.png)
![image](https://user-images.githubusercontent.com/62196528/235356328-1c0034ab-a49f-4635-8f23-5227f84450c3.png)|![image](https://user-images.githubusercontent.com/62196528/235356910-9884cf4c-d4ba-458c-ac39-0992a9d49085.png)
![image](https://user-images.githubusercontent.com/62196528/235356367-8911ee53-efc6-4e18-96e9-8c15f0be8c8c.png)|![image](https://user-images.githubusercontent.com/62196528/235356889-4aa9ac93-cf1a-47c7-9324-6e31b04e03ea.png)
